src/attached_probe.cpp

```diff
@@ -601,26 +601,25 @@ Result<std::unique_ptr<AttachedKprobeProbe>> AttachedKprobeProbe::make(
   if (!is_symbol_kprobe)
     funcname += probe.attach_point;
 
-  LOG(V1) << "bpf_attach_kprobe(" << prog.fd() << ", " << probe.type << ", "
-          << eventname(probe, offset) << ", " << funcname << ", " << offset
-          << ", 0)";
-  int perf_event_fd = bpf_attach_kprobe(prog.fd(),
-                                        attachtype(probe.type),
-                                        eventname(probe, offset).c_str(),
-                                        funcname.c_str(),
-                                        offset,
-                                        0);
+  struct bpf_kprobe_opts opts;
+  ::memset(&opts, 0, sizeof(opts));
+  opts.sz = sizeof(struct bpf_kprobe_opts);
```

Comment:
I know I'm late to the party but why don't we use `BPFTRACE_LIBBPF_OPTS` which we have for that purpose?

Reply:
Ah yes. Forgot about that!

Reply:
#4209